webpack.dev.js

```diff
@@ -0,0 +1,7 @@
+const merge = require('webpack-merge');
+const common = require('./webpack.common.js');
+
+module.exports = merge(common, {
+    mode: 'development',
+    devtool: 'inline-source-map'
+});
```

Comment:
Can you add custom `devServer` configuration?
For example, to set custom port and automatically open application in browser on start.